packages/react-native-editor/ios/GutenbergDemo/GutenbergViewController.swift

```diff
@@ -419,6 +430,26 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
     }
 
     func gutenbergEditorSettings() -> GutenbergEditorSettings? {
+        guard ProcessInfo.processInfo.arguments.count >= 3 else {
+            return nil
+        }
+        
+        let initialProps = ProcessInfo.processInfo.arguments[2]
+
+        if let data = initialProps.data(using: .utf8) {
+            do {
+                if let json = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any] {
+                    let settings = EditorSettings()
+
+                    settings.rawStyles = json["rawStyles"] as? String
+                    settings.rawFeatures = json["rawFeatures"] as? String
+                    return settings
+                }
+            } catch {
+                print(error.localizedDescription)
+            }
+        }
+        
```

Comment:
The initial props are processed in the function `getInitialPropsFromArgs`, so we could simplify this part by using that function. You can see an example in the function `gutenbergInitialContent`.

https://github.com/WordPress/gutenberg/blob/35573912f353c8125ff060f9af67df76cf890144/packages/react-native-editor/ios/GutenbergDemo/GutenbergViewController.swift#L390-L395

Reply:
Thanks for the suggestion! I will update it.